core/src/main/java/nu/yona/server/analysis/entities/DayActivity.java

```diff
@@ -144,22 +139,22 @@ private int getDurationInSpreadItem(ZonedDateTime startTime, ZonedDateTime endTi
 	{
 		if (spreadStartIndex == spreadEndIndex)
 		{
-			// partial span
-			return (int) startTime.until(endTime, ChronoUnit.MINUTES) + 1;
+			// partial span of spread item
+			return (int) startTime.until(endTime, ChronoUnit.MINUTES);
 		}
 		else if (spreadItemIndex == spreadStartIndex)
 		{
-			// start part
+			// start spread item
```

Comment:
Here I meant that the start of the activity occurs somewhere in the spread item. So the spanning is here from somewhere in the spread item to the end. A better comment is `// activity starts somewhere in spread item and continues till the end`.